__tests__/input.spec.js

```diff
@@ -306,4 +340,25 @@ describe('Questions of type input, password and number', () => {
     expect(wrapper.vm.questions[1].shouldShow).toBe(false);
     expect(wrapper.vm.getIssues()).toBe(undefined);
   });
+
+  test('Input with hint', async () => {
+    const wrapper = mount(Form, { });
+    wrapper.setProps({ questions: questionInputHint });
+    await Vue.nextTick();
+    await utils.sleep(300);
+
+    const labels = wrapper.findAll('p.question-label');
+
+    expect(labels.at(0).findAll('span.error-validation-asterisk').at(0).element.innerHTML).toBe('*');
+    expect(labels.at(0).findAll('span.question-message').at(0).element.innerHTML).toBe(questionInputHint[0].message);
+    expect(labels.at(0).findAll('span.question-hint').at(0).element.innerHTML).toContain('v-tooltip');
```

Comment:
Use [vue test component selectors](https://vue-test-utils.vuejs.org/api/selectors.html#vue-components) to test the value of the tooltip